ocm-container.sh

```diff
@@ -70,14 +71,39 @@ then
   SSH_AGENT_MOUNT="--mount type=bind,src=/run/host-services/ssh-auth.sock,target=/tmp/ssh.sock,readonly"
 fi
 
+### Kerberos Ticket Mounting
+OCM_CONTAINER_KRB5CC_FILE=${KRB5CCNAME:-/tmp/krb5cc_$UID}
+if [ -f $OCM_CONTAINER_KRB5CC_FILE ]
+then
+  KRB5CCFILEMOUNT="-v ${OCM_CONTAINER_KRB5CC_FILE}:/tmp/krb5cc:ro"
+fi
+
+### Kerberos User Override
+if [ -z $OCM_CONTAINER_KERBEROS_USER ]
+then
+  OCM_CONTAINER_KERBEROS_USER=$(whoami)
+fi
+USER=$OCM_CONTAINER_KERBEROS_USER
+
+### Automatic Login Detection
+if [ -n "$ARGS" ]
+then
+  INITIAL_CLUSTER_LOGIN="-e INITIAL_CLUSTER_LOGIN=$ARGS"
+fi
+
 ### start container
 ${CONTAINER_SUBSYS} run -it --rm --privileged \
--e "OCM_URL=${OCM_URL}" \
+-e "OCM_URL" \
+-e "USER" \
 -e "SSH_AUTH_SOCK=/tmp/ssh.sock" \
+-e "KRB5CCNAME=/tmp/krb5cc" \
+-e "OFFLINE_ACCESS_TOKEN" \
```

Comment:
These need to be set here so that this can become scriptable, as ~.bashrc is not run when the container inits without a TTY.